### ISSUE TITLE
Support for Berlin hard-fork

### ISSUE THREAD

**Issue by drortirosh** [ethereum/web3.js#3963]
With Berlin hard-fork [announced](https://blog.ethereum.org/2021/03/08/ethereum-berlin-upgrade-announcement/) to be released at 14-Apr, it would be nice if the web3 would support sending (and decoding) [EIP-2918](https://eips.ethereum.org/EIPS/eip-2718)/[EIP-2930](https://eips.ethereum.org/EIPS/eip-2930) transactions.

**Comment by Spacesai1or** [comment]
Hi there, PR #3919 aims to add support for EIP2918 (and might add support for EIP2930, if not we'll just open a separate PR to handle any needed changes), we're just waiting on [this](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1048) to be included in a release of `@ethereumjs.tx`

Thank you :)

**Comment by holgerd77** [comment]
Hi hi, Berlin releases for `@ethereumjs/tx` and other EthereumJS libraries like the VM are prepared here https://github.com/ethereumjs/ethereumjs-monorepo/pull/1151 and will be released next Tuesday (if nothing unexpected comes in between).

**Comment by cryptoDevTrader** [comment]
Any update on this? https://github.com/ethereumjs/ethereumjs-monorepo/pull/1151 was merged 17 days ago.
